### PR TITLE
Import XML Doc comments for System.Runtime

### PR DESCRIPTION
**Pull request by mairaw**
/cc @cartermp @Priya91 

A couple of things I've noticed with this one:
1. The two files contain duplicate classes. Should we remove the comments for the class from the Manual.cs file when that happens? Otherwise, how docfx will know from where to grab the comment? We don't want to update multiple locations when we find a doc bug.
2. In a couple of exceptions, we mention Windows 98 and Windows Millenium. Maybe a doc bug since this is not relevant anymore?
3. We have a couple of exceptions where we mention specific .NET Framework versions (see WaitAll for example). Not sure which one is thrown by Core since I couldn't find the actual source here. (And we also say in the docs that older versions throw ApplicationException when the code seems to throw ArgumentException. Another doc bug?)

### CODE REVIEW

File: src/System.Runtime/ref/System.Runtime.Manual.cs [hunk 1]
```diff
@@ -16,29 +16,88 @@
...
         public const float NaN = (float)0.0 / (float)0.0;
     }
+    /// <summary>
+    /// Represents a double-precision floating-point number.
+    /// </summary>
     public partial struct Double
     {
+        /// <summary>
+        /// Represents the smallest possible value of a <see cref="Double" />. This field is
+        /// constant.
```
Comment (stephentoub) [inline src/System.Runtime/ref/System.Runtime.Manual.cs]: I've always found it a bit strange that we say things like "This field is constant".  While this is technically fine, I'm not sure most developers think of consts as being fields.

File: src/System.Runtime/ref/System.Runtime.Manual.cs [hunk 2]
```diff
@@ -16,29 +16,88 @@
...
+        /// </summary>
         public const double PositiveInfinity = (double)1.0 / (double)(0.0);
+        /// <summary>
+        /// Represents a value that is not a number (NaN). This field is constant.
+        /// </summary>
         public const double NaN = (double)0.0 / (double)0.0;
     }
+    /// <summary>
+    /// Represents type declarations: class types, interface types, array types, value types, enumeration
+    /// types, type parameters, generic type definitions, and open or closed constructed generic types.To
```
Comment (stephentoub) [inline src/System.Runtime/ref/System.Runtime.Manual.cs]: Nit: missing a space before the To

**Comment by stephentoub** [comment]
I spot-checked this and it generally looks fine, though it'd be nice to incorporate some comments made on other PRs, e.g. using ```{T}``` instead of ```'1```, though I wouldn't block on that.  As long as this all compiles and including XML comments in our ref files is the path we're following, I'd suggest merging it and making incremental changes as issues are discovered, fixes needed, etc.  I do still think it'd be good enable doc comment building in the ref .csproj as part of this, so that we know the comments we're checking in actually do compile (otherwise they're not useful at all).

**Comment by cartermp** [comment]
Closing this out as we have a different plan for reference docs now.

**Pull request closed without merging**